### DISCUSSION TITLE
Next release 1.35.2

### DISCUSSION THREAD

**Discussion by Beer17HWAM** [Koenkk/zigbee2mqtt#21099] [General]
Can anyone explain why the next release 1.35.2 is taken so long? How does a "Pull request" work? I'm not a github expert so I don't know how the proces for a new release is working. I know there are a lot of things changed in the new release and I guess it's ready but it's not released. Please explain.

**Comment by Nardol** [comment]
New version is published on first of each month, except in particular circumstance.
